tests/generic_config_updater/test_incremental_qos.py

```diff
@@ -11,7 +11,7 @@
 from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
 
 pytestmark = [
-    pytest.mark.topology('t0', 'm0'),
+    pytest.mark.topology('t0', 'm0', 'mx'),
```

Comment:
This is related to QoS? We might not need this for both m0 and mx. Need check it

Reply:
Fixed